src/Katas/question5/exercise1.js

```diff
@@ -0,0 +1,9 @@
+function decimalToBinary(decimal) {
+  if (typeof decimal !== "number") {
+    return "Not a valid number.";
+  }
```

Comment:
I would prefer to throw an error instead of returning a string in a function that should return a number. But in this case, both ways are equally good in this case.